nodebuilder/init_test.go

```diff
@@ -63,9 +64,9 @@ func TestInitErrForLockedDir(t *testing.T) {
 	defer flock.Unlock() //nolint:errcheck
 	nodes := []node.Type{node.Light, node.Bridge}
 
-	for _, node := range nodes {
-		cfg := DefaultConfig(node)
-		require.Error(t, Init(*cfg, dir, node))
+	for _, currentNode := range nodes {
+		cfg := DefaultConfig(currentNode)
+		require.Error(t, Init(*cfg, dir, currentNode))
```

Comment:
?

Reply:
Same for other renamings, let's avoid them

Reply:
ok, let me follow the naming convention, I wasn't aware of it before. thank you for reminding me. I will try to fix the issues next Monday.

Reply:
When I try to change it back to 'node', I find it collides with an imported package name. Do you have any suggestions?